frontend/src/components/pki-syncs/forms/PkiSyncConnectionField.tsx

```diff
@@ -47,43 +69,62 @@ export const PkiSyncConnectionField = ({ onChange: callback }: Props) => {
       <Controller
         render={({ field: { value, onChange }, fieldState: { error } }) => (
           <FormControl
+            tooltipText="App Connections can be created from the Organization Settings page."
```

Comment:
I think we should default to pointing them to the project app connections instead of at the organization-level?